test/manual/definitions/apm/.net/linux/linux2-aspnetcore5.json

```diff
@@ -0,0 +1,51 @@
+{
+  "global_tags": {
+    "owning_team": "OpenSource",
+    "Environment": "development",
+    "Department": "Product",
+    "Product": "Virtuoso"
+  },
+  "resources": [{
+      "id": "host1",
+      "display_name": "AwsLinux2ASPNETCore",
+      "provider": "aws",
+      "type": "ec2",
+      "size": "t3.nano",
+      "ami_name": "amazonlinux-2-base*",
+      "user_name": "ec2-user"
+    }],
+  "services": [{
+      "id": "aspnetcore",
+      "destinations": [ "host1" ],
+      "source_repository": "https://github.com/newrelic/open-install-library.git",
+      "deploy_script_path": "test/deploy/linux/.net/install/rhel/roles",
+      "port": 9999
+    },
+    {
+      "id": "apache1",
+      "destinations": [ "host1" ],
+      "source_repository": "https://github.com/newrelic/open-install-library.git",
+      "deploy_script_path": "test/deploy/linux/apache/install/rhel/roles",
+      "port": 80
+    },
+    {
+      "id": "dotnet1",
+      "destinations": [ "host1" ],
+      "source_repository": "https://github.com/newrelic/open-install-library.git",
+      "deploy_script_path": "test/deploy/linux/apache/deploy-application/.net/rhel/roles",
+      "port": 8080,
+      "params": {
+        "is_selfcontained": "true",
+        "use_executable": "true",
+        "proxy_port": "5001"
+      }
+    },
+    {
+      "id": "dotnet2",
+      "destinations": [ "host1" ],
+      "source_repository": "https://github.com/newrelic/open-install-library.git",
+      "deploy_script_path": "test/deploy/linux/apache/deploy-application/.net/rhel/roles",
+      "port": 8081,
+      "proxy_port": "5000"

```

Comment:
proxy_port should be within a "params" element

Reply:
fixed